website/addons/dryad/tests/test_models.py

```diff
@@ -0,0 +1,45 @@
+from nose.tools import *  # noqa
+
+from framework.auth import Auth
+
+from website.addons.dryad.tests.utils import *
+from website.addons.dryad.utils import DryadRepository
+
+
+class DryadTestModel(DryadTestCase):
```

Comment:
If you can get this working with the generic models, try to also use use the generic test mixins found in `website.addons.base.testing` -- they might require some overrides, but will give a large number of tests for free.